ldap/servers/slapd/tools/dbscan.c

```diff
@@ -1288,12 +1288,6 @@ main(int argc, char **argv)
     }
 
 done:
-    if (key.data) {
-        free(key.data);
-    }
-    if (data.data) {
-        free(data.data);
-    }
```

Comment:
The fix looks good, just to clarify my understanding is the problem that key.data and data.data were not set to NULL after 'free' ? So next dblayer_value_free triggered a double free ?

Reply:
dblayer_value_free() is basically redundant of those frees I removed.  We don't need both...